src/app_service/service/chat/dto/chat.nim

```diff
@@ -12,6 +14,11 @@ type ChatType* {.pure.}= enum
   Profile = 4,
   CommunityChat = 6
 
+type ChannelGroupType* {.pure.}= enum
+  Unknown = "unknown",
```

Comment:
@jrainville I think that we shouldn't have "unknown" ever, just checking for "personal" or "community" all other options are ignored.

Reply:
I'm just mapping to what status-go gives. Maybe one day we'll have a warning or error for them. Or we just will ignore those in the front-end